Rest/alm-vss-client/src/main/java/com/microsoft/alm/client/VssRestClientHandlerBase.java

```diff
@@ -0,0 +1,191 @@
+package com.microsoft.alm.client;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.UUID;
+
+import com.microsoft.alm.client.utils.StringUtil;
+import com.microsoft.alm.visualstudio.services.webapi.ApiResourceLocation;
+import com.microsoft.alm.visualstudio.services.webapi.ApiResourceLocationCollection;
+import com.microsoft.alm.visualstudio.services.webapi.ApiResourceVersion;
+
+public abstract class VssRestClientHandlerBase implements VssRestClientHandler {
```

Comment:
This base class provides more "public" API than the interface defined -- which means it's hard to program against the interface.  Consider declare all public API in the interface?  Or create a separate interface to declare those APIs if they don't logically belong in the same group?